src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function convertNumber(input) {
+  const convertedNumber = +input.slice(1).replace(',', '');
```

Comment:
The `convertNumber` function assumes the input is a string starting with a currency symbol followed by a number with commas. Ensure the input format matches this expectation.